app/code/Magento/ImportExport/Controller/Adminhtml/Import/Start.php

```diff
@@ -93,6 +93,20 @@ public function execute()
                 $this->addErrorMessages($resultBlock, $errorAggregator);
             } else {
                 $this->importModel->invalidateIndex();
+
+                $noticeHtml = $this->historyModel->getSummary();
+
+                if($this->historyModel->getErrorFile()) {
```

Comment:
```suggestion
                if ($this->historyModel->getErrorFile()) {
```

We should have an extra space after the if keyword to match our coding standards.